packages/validation/src/nodebalancers.schema.ts

```diff
@@ -352,32 +374,45 @@ export const NodeBalancerSchema = object({
 
   configs: array()
     .of(createNodeBalancerConfigSchema)
-    /* @todo there must be an easier way */
-    .test('unique', 'Port must be unique.', function (value) {
-      if (!value) {
+    .test('unique', 'Port must be unique.', function (configs) {
+      if (!configs) {
         return true;
       }
-      const ports: number[] = [];
-      const configs = value.reduce((prev: number[], value, idx: number) => {
-        if (!value.port) {
-          return prev;
-        }
-        if (!ports.includes(value.port)) {
-          ports.push(value.port);
-          return prev;
+      const indexsOfConfigsWithNonUniquePort: number[] = [];
```

Comment:
Nit: typo throughout this file with this variable name.

```suggestion
      const indexesOfConfigsWithNonUniquePort: number[] = [];
```
or `indices`. 

TIL! 
<img width="670" height="275" alt="image" src="https://github.com/user-attachments/assets/3ef0b5c0-8cb8-4546-939a-8e6e9f908f91" />

Reply:
Thanks! ✨  I also had no idea